lambdanote/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App />, div);
```

Comment:
This is throwing an error for me in my preview -- it's not necessary but you may wish to delete the `App.test.js` file just in case it poses problems.